torch/csrc/jit/passes/python_print.h

```diff
@@ -12,33 +12,21 @@ struct Method;
 struct Module;
 } // namespace script
 
-TORCH_API void PythonPrint(
-    std::ostream& out,
-    const Graph& graph,
-    std::vector<at::Tensor>& tensor_table,
-    std::vector<ClassTypePtr>& class_table,
-    bool enforce_importable = false);
-
-TORCH_API void PythonPrint(
-    std::ostream& out,
-    const script::Method& graph,
-    std::vector<at::Tensor>& tensor_table,
-    std::vector<ClassTypePtr>& class_table,
-    bool enforce_importable = false);
-
 TORCH_API void PythonPrint(
```

Comment:
not for this PR, but I feel like this signature is getting perilously large with a lot of communication going on between the exporter and PythonPrint in the form of various in-out params. We're not enforcing a clean abstraction boundary here.